attack_range.conf

```diff
@@ -306,6 +346,26 @@ windows_client_os = import-ami-0e5a092cc20dbb992
 # for mode terraform specify the ami name
 # instructions to upload your own Windows 10 ami can be found in the wiki
 
-windows_client_join_domain = 1
+windows_client_join_domain = 0
 # specify if the windows client should join the windows domain
 # possible values: 1, 0
+
+windows_client_zeek_capture = 0
+#specify if you want to capture wire data from the Windows client
+# possible values: 1, 0
+
+[zeek_sensor]
+# customize the zeek sensor
+
+zeek_sensor_instance_type = m5.2xlarge
+# specify the instance type for the zeek sensor
+# you must use an AWS Nitro instance type (m5.2xlarge)
+
+splunk_uf_linux_deb_url = https://download.splunk.com/products/universalforwarder/releases/8.0.5/linux/splunkforwarder-8.0.5-a1a6394cc5ae-linux-2.6-amd64.deb
+# Specify the download URL of the Splunk Linux (debian based) universal forwarder
+
+splunk_uf_binary = splunkforwarder-8.0.5-a1a6394cc5ae-linux-2.6-amd64.deb
```

Comment:
Not sure if we want this in the configuration.

Reply:
This is also needed for the UF that runs on the Zeek sensor